docs/application/web/guides/data/data-filter.md

```diff
@@ -351,44 +351,44 @@ The following table lists the filter attributes related to the `findMessage()` m
-| `messageStatus`  | No                         | No                               |
+| `inResponseTo`   | Yes                        | No                               |
+| `messageStatus`  | Yes                        | No                               |
 | `attachments`    | No                         | No                               |
 
-The following table lists the filter attributes related to the `findConversations()` method.
+The following table lists the attributes by which `MessageConversation` objects may be filtered.
 
-**Table: Filter attributes for finding conversations**
+<span id="conversation-filters-table">**Table: Filter attributes for finding conversations**</span>
 
 | Attribute        | Attribute filter supported | Attribute range filter supported |
 | ---------------- | -------------------------- | -------------------------------- |
 | `id`             | Yes                        | No                               |
 | `type`           | Yes                        | No                               |
 | `timestamp`      | No                         | Yes                              |
-| `messageCount`   | Yes                        | No                               |
-| `unreadMessages` | Yes                        | No                               |
+| `messageCount`   | Yes                        | Yes                              |
+| `unreadMessages` | Yes                        | Yes                              |
 | `preview`        | Yes                        | No                               |
-| `subject`        | No                         | No                               |
-| `isRead`         | No                         | No                               |
+| `subject`        | Yes                        | No                               |
+| `isRead`         | Yes                        | No                               |
 | `from`           | Yes                        | No                               |
 | `to`             | Yes                        | No                               |
-| `cc`             | No                         | No                               |
-| `bcc`            | No                         | No                               |
-| `lastMessageId`  | No                         | No                               |
+| `cc`             | Yes                        | No                               |
+| `bcc`            | Yes                        | No                               |
+| `lastMessageId`  | Yes                        | No                               |
 
-The following table lists the filter attributes related to the `findFolders()` method.
+The following table lists the attributes by which `MessageFolder` objects may be filtered.
 
-**Table: Filter attributes for finding message folders**
+<span id="folder-filters-table">**Table: Filter attributes for finding folders**</span>
```

Comment:
When you update this PR, please remove this span tag.  
Only **\*\*Table: Filter attributes for finding folders\*\*** is enough.

Reply:
If you want to create a link to a point which is not a title, you can use `<a name="folder-filters-table"></a>`

For example, if you check web/guides/multimedia/media-controller.md file, 
you'll see "send_custom_commands" is inserted as below to jump to the one of list item.

![image](https://user-images.githubusercontent.com/9958374/52384594-80d4df80-2ac1-11e9-9848-a31d6e8951b4.png)

Would you please use `<a name>` to link to the tables?

Reply:
I have applied the suggested change - anchors are now added with <a name="..."></a>.